CONTRIBUTING.md

```diff
@@ -32,17 +32,33 @@ Add some tests and make your change. Re-run the tests with:
 yarn test
 ```
 
+## Emoji commits
+
+We have foudn that prefacing a commit message or PR title with an emoji can be a great way to improve the developer experience when browsing the repo code. Additionally, it is a terse way to convey information. Many of our contributors have found the guide at https://gitmoji.carloscuesta.me/ to be helpful in preservign this dynamic.
+
 ## Documentation
 
 If your change affects the public API of any packages within Quilt (i.e. adding or
 changing arguments to a function, adding a new function, changing the
 return value, etc), please ensure the documentation is also updated to
 reflect this. Documentation is in the `README.md` files of each package. If further documentation is needed please communicate via Github Issues.
 
-## Changelog
+## Testing
+
+The packages in Quilt are used in mission-critical production scenarios. As such, we try not to merge any untested code. The coverage doesn't strictly need to be 100% across the board, but testing should remain a primary concern.
+
+## TODO Comments
 
-Please make sure you update `CHANGELOG.md` with a high-level description of any changes you make. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+TODO comments may seem like a great placeholder for work in progress. We prefer top handle this in a different way, using a combination of feature branches and github issues.
+
+### Follow-up Github issues
+If your changes are complete in functionality, but you're not quite happy with auxilliary things liek documentation or testing, then feel free to make a github issue to track the work that needs to be done. These issues should be linked in the PRs that needa bit more work. This will allow context to be drawn from the code in a more trackable way than a TODO comment. Also, it allows the PR reviewers to see that the documentation rot esting is purposefully incompleteand that an appropriate issue exists to track the follow-up work.
+
+### Feature branches
+Another option, if you'd like to break work down into reviewable chunks, is to use a feature branch. This would be an initially empty branch that contains the entirety of your feature. Additional units of work can be distributed across several PRs into the feature branch, merged independently, and then the feature branch can be merged as a complete unit into the master branch, when it's ready.
 
 ## Releasing a new version
 
-The `yarn version` command can be used to create a new version. This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```

Comment:
this is a carbon copy of another repo; we don't use this process for releasing a new version. Furthermore, we don't want any non- repo-owners to attempt a release, as this will cause some problems in our current setup